packages/builder/test/helpers/params.ts

```diff
@@ -0,0 +1,26 @@
+import { AlgobRuntimeEnv, PromiseAny } from "../../src/types";
+
+export function mkAlgoEnv(): AlgobRuntimeEnv {
```

Comment:
let's rename to `mkAlgobEnv` (notice `b`)

Reply:
To what?

Reply:
you don't have `b` in `mkAlgoEnv` name

Reply:
ok